data/i18n/es/es.toml

```diff
@@ -1,5 +1,11 @@
 # i18n strings for the Spanish (main) site.
 
+[docs_version_latest_heading]
+other = "Versión Actual"
+
+[docs_version_other_heading]
+other = "Versiones anteriores"
```

Comment:
We also need `docs_version_current` (I missed this in my changes for French)

Reply:
Try: “(versión actual)”

Reply:
Thank you, done.